pkg/tls/ca.go

```diff
@@ -11,6 +12,12 @@ import (
 	"time"
 )
 
+// Supported TLS key types.
+const (
+	KeyTypeRSA   = "rsa"
+	KeyTypeECDSA = "ecdsa"
+)
+
```

Comment:
These constants are no longer used and can be removed.

Reply:
oops, thanks!